narwhals/dataframe.py

```diff
@@ -857,28 +877,47 @@ def drop(self, *columns: str | Iterable[str]) -> Self:
             *columns: Names of the columns that should be removed from the dataframe.
 
         Examples:
-            >>> import pandas as pd
+            Drop a single column by passing the name of that column.
+
             >>> import polars as pl
+            >>> import pandas as pd
             >>> import narwhals as nw
-            >>> df = {"foo": [1, 2, 3], "bar": [6.0, 7.0, 8.0], "ham": ["a", "b", "c"]}
-            >>> df_pd = pd.DataFrame(df)
-            >>> df_pl = pl.DataFrame(df)
 
-            We define a library agnostic function:
+            >>> # Example: Using Polars
```

Comment:
why change this? we want the docstrings to follow the template of:

1. create `df_pd` and `df_pl`
2. "we define a library-agnostic function"
3. show that you can pass either pandas or polars to your function